notebooks/02_quant_activation_overview.ipynb

```diff
@@ -315,16 +326,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 10,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "True\n",
+      "tensor(True)\n",
+      "tensor(True)\n",
+      "tensor(True)\n"
+     ]
+    }
+   ],
    "source": [
     "return_disabled_quant_relu = QuantReLU(act_quant=None, return_quant_tensor=True)\n",
     "relu_out_tensor = return_disabled_quant_relu(out_tensor)\n",
-    "assert relu_out_tensor.is_valid==True\n",
-    "assert relu_out_tensor.scale == out_tensor.scale\n",
-    "assert relu_out_tensor.zero_point == out_tensor.zero_point\n",
-    "assert relu_out_tensor.bit_width == out_tensor.bit_width"
+    "assert_with_message((relu_out_tensor.is_valid==True))\n",
```

Comment:
Double parenthesis

Reply:
Also, let's remove `assert condition == True`, and change it to `assert condition`.
Similar with `assert condition == False` that becomes `assert not condition`
This line becomes :
```suggestion
    "assert_with_message(relu_out_tensor.is_valid)\n",
```
I think there are other instances around the notebooks